src/LuaEngine.cpp

```diff
@@ -1105,6 +1113,7 @@ void LuaEngine::Register()
 		{ "CameraSpaceToScreenSpace", l_engine_camera_space_to_screen_space },
 		{ "WorldSpaceToScreenSpace",     l_engine_world_space_to_screen_space },
 		{ "WorldSpaceToShipSpace",     l_engine_world_space_to_ship_space },
+		{ "GetLoadOption", l_get_load_option },
```

Comment:
Not aimed at this contribution, but this module is becoming way too much of a kitchen sink.

Reply:
Where it should be? Game.lua? Im thought about it, and selected Engine as best choice for me)